datafusion/core/Cargo.toml

```diff
@@ -117,10 +117,6 @@ name = "scalar"
 harness = false
 name = "physical_plan"
 
-[[bench]]
-harness = false
-name = "parquet_query_sql"
-
 [[bench]]
```

Comment:
Do you think the  scheduler should be an optional `feature` of the `core` (eventually?)

Reply:
That's the goal 😃